app/recruitment/[id]/_components/comment/form.tsx

```diff
@@ -0,0 +1,45 @@
+'use client';
+
+import { useRef } from 'react';
+
+import { Button } from '~/components/button';
+import { Label } from '~/components/label';
+import { Textarea } from '~/components/textarea';
+import useMutationComment from '~/hooks/mutations/useMutationComment';
+
+interface CommentFormProps {
+  accompanyPostId: string;
+}
+
+const CommentForm = ({ accompanyPostId }: CommentFormProps) => {
+  const ref = useRef<HTMLTextAreaElement>(null);
+  const { mutate } = useMutationComment(accompanyPostId);
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    if (!ref.current || !ref.current.value) {
+      return;
+    }
+
+    mutate({ userId: '1', content: ref.current.value });
+    ref.current.value = '';
+  };
+
+  return (
+    <form onSubmit={handleSubmit} className="flex flex-col gap-7">
+      <Label htmlFor="comment" />
+      <Textarea
+        className="h-20 resize-none"
+        ref={ref}
+        id="comment"
+        name="comment"
+        placeholder="댓글을 입력하세요"
+      />
+      <Button className="w-14 self-end" type="submit">
+        등록
+      </Button>
```

Comment:
api 요청 중에 버튼을 여러번 클릭하게 된다면 중복 호출이 가능할 것 같아서, 필요하다면 useMutationComment로 반환된 isLoading으로 handleSubmit을 한번만 제출 할 수도 있을까요?
```suggestion
      <Button className="w-14 self-end" type="submit" disabled={isLoading}>
        등록
      </Button>
```